### PR TITLE
Multipage

### LINKED ISSUE #5: allow creating multi-page PDFs
IIUC the package currently aims to create one PDF file per page in the output file group.

It would be a thrill to have an option to concatenate all pages into one multi-page PDF file (with proper PDF meta-data about the physical page numbers).

### PR DESCRIPTION
**Pull request by JKamlah**
Summary:
- Add ocrd-logger
- Add the multipage refs to mets 
- Cleaning pdfmarks after use (with option to keep it)
- Add automated metadata creation 
- Add 'pagelabel' option to switch between different pagelabel, e.g. to consecutively numbers 
- Add 'overwrite' option to delete pre-existings files
- Update README
- Some little fixes e.g. switching open-jdk-version in Dockerfile and some minor rearrangments

### CODE REVIEW

File: ptp/multipagepdf.py [hunk 1]
```diff
@@ -4,49 +4,80 @@
...
+def get_metadata(mets):
+    title = mets._tree.getroot().find('.//mods:title', NS)
+    subtitle = mets._tree.getroot().find('.//mods:subtitle', NS)
+    title = title.text if title is not None else ""
+    title += "Subtitle: "+subtitle.text if subtitle else ""
+    publisher = mets._tree.getroot().find('.//mods:publisher', NS)
+    author = mets._tree.getroot().find('.//mods:creator', NS)
+    return {'Author':author.text if author is not None else "",
+            'Title': title,
+            'Keywords': publisher.text+" (Publisher)" if publisher is not None else ""}
```
Comment (bertsky) [inline ptp/multipagepdf.py:22]: Nice!

@kba @tboenig Is this already comparable to what DFGViewer provides in its PDF export?

**Review by bertsky — COMMENTED** [review]
Splendid!

File: ocrd-pagetopdf [hunk 2]
```diff
@@ -40,6 +40,19 @@ main () {
         warning "Without a second input file group for images, the original imageFilename will be used"
     fi
     out_file_grp=${ocrd__argv[output_file_grp]}
+    
+    # Remove pre-existing files
+    if [ -d ${out_file_grp} ]; then
+        case ${params['overwrite']} in
```
Comment (kba) [inline ocrd-pagetopdf:46]: `--overwrite` is now part of the [CLI processor spec](https://ocr-d.de/en/spec/cli#--overwrite). bashlib support is not there yet, unfortunately https://github.com/OCR-D/core/issues/476

**Review by kba — COMMENTED** [review]
Forgot to watch this repo, obviously, watching now. This is great stuff @JKamlah!

**Pull request merged**